plugin/evm/header/extra_test.go

```diff
@@ -610,102 +610,37 @@ func TestVerifyExtra(t *testing.T) {
 	}
 }
 
-func TestPredicateBytesFromExtra(t *testing.T) {
+func TestPredicateBytesExtra(t *testing.T) {
```

Comment:
Why is this test being overhauled in a PR that's supposed to be limited to "small changes to the libevm branch"? The same goes for deletion of `TestSetPredicateBytesInExtra`.

This is way too significant a change to just lump in like this because it risks hiding a correlated break somewhere else that the change in the test no longer picks up.

Reply:
That was a suggestion made by @ceyonur:

> On the side note should this [test](https://github.com/ava-labs/coreth/pull/828/commits/21163f1808c07ee81e7eb07e17538971ccef75de) check against PredicateBytesFromExtra  to make sure they're inverse of each other?

and was not included in the (merge master) PR to avoid having to rebase + re-review everything. I can split it in another PR if you feel it should be separate, fine with me as well.

Reply:
Re-added previous tests, and made this new `TestPredicateBytesExtra` test just an addition on of of the two tests.

Reply:
Thanks. I'm not disagreeing with Cey's suggestion; just saying it should be standalone from a review-safety perspective. I'm very happy with adding new tests (as you've done it).